crates/bevy_winit/src/converters.rs

```diff
@@ -266,12 +266,3 @@ pub fn convert_cursor_icon(cursor_icon: CursorIcon) -> winit::window::CursorIcon
         CursorIcon::RowResize => winit::window::CursorIcon::RowResize,
     }
 }
-
-/// Map [`bevy_window::CursorGrabMode`] to [`winit::window::CursorGrabMode`].
-pub fn convert_cursor_grab_mode(mode: CursorGrabMode) -> winit::window::CursorGrabMode {
```

Comment:
This is non-blocking but I'm going to Chesterton-fence you here, but what purpose did this serve? I notice it is marked `pub` so technically this is breaking change. Though such a conversion method defined as a free function is really weird.

Reply:
This is only for bevy crates' internal uses.  I introduced this function for `bevy_window` abstracts its own `CursorGrabMode` struct because `bevy_window` doesn't want to depend on `winit`, then `bevy_winit` should convert `bevy_window`'s `CursorGrabMode` to `winit::window::CursorGrabMode`. I think this function shouldn't be `pub` but I just followed these conversion functions' signature above. Anyway, we could remove it now and it shouldn't break much code since it's not for end users in the beginning. If this breaks the semantic version, please help us find a way to get of it... Sorry for this mistake.

Reply:
Don't worry. This is perfectly fine.